docs/en/latest/plugins/prometheus.md

```diff
@@ -102,6 +102,9 @@ plugin_attr:
 
 You can then expose it by using the [public-api](public-api.md) Plugin.
 
+Note:  If the Prometheus plugin collects too many metrics, it will take CPU resources to calculate the metric data when getting the metrics via URI, which may affect APISIX to process normal requests. To solve this problem, APISIX exposes the URI and calculates the metrics in the privileged agent.
```

Comment:
More note: This feature requires users to use `apisix-base` instead of the official version of openresty.

Reply:
done